tutorials/beginner/index.md

```diff
@@ -1,13 +1,37 @@
 ---
 layout: tutorial-beginner
+title: Beginner Tutorial
 ---
 
 {% include base.html %}
 
 # Overview
 
+This beginner tutorial covers the basic use of openHAB. It is following a step-by-step approach and the goal is to enable a new user to perform some first configuration tasks. It is based on the [demo installation](http://docs.openhab.org/tutorials/demo.html#choosing-demo-as-a-base-package-and-further-add-ons-to-be-installed) and will add more functionality offered by the [System Info](http://docs.openhab.org/addons/bindings/systeminfo/readme.html) binding.
 
-This part covers all the basic use of openHAB: how to install a binding, how to configure them, the inbox....
+The result of the additional configuration will look as follows:
 
-The goal is to cover a first level of usage with the example of a simple binding: the _System info_ binding. This binding System provides operating system and hardware information.
+* A section with performance metrics
+* An alarm functionality, which is constantly monitoring one of the metrics
+* A diagram showing the historical values of the metrics
 
+![](images/Overview_Result.png)
+
+
+The tutorial is divided in five parts:
```

Comment:
five is confusing here, I see three headings and three bullet points.
Yes, there are five links to sub-articles but you know what I mean!?